include/ur_api.h

```diff
@@ -133,137 +133,171 @@ typedef struct ur_mem_handle_t_ *ur_mem_handle_t;
+        43, ///< [Validation] generic error code for unsupported versions
+    UR_RESULT_ERROR_UNSUPPORTED_FEATURE =
+        44, ///< [Validation] generic error code for unsupported features
+    UR_RESULT_ERROR_INVALID_ARGUMENT =
+        45, ///< [Validation] generic error code for invalid arguments
+    UR_RESULT_ERROR_INVALID_NULL_HANDLE =
+        46, ///< [Validation] handle argument is not valid
+    UR_RESULT_ERROR_HANDLE_OBJECT_IN_USE =
+        47, ///< [Validation] object pointed to by handle still in-use by device
+    UR_RESULT_ERROR_INVALID_NULL_POINTER =
+        48, ///< [Validation] pointer argument may not be nullptr
+    UR_RESULT_ERROR_INVALID_SIZE =
+        49, ///< [Validation] size argument is invalid (e.g., must not be zero)
+    UR_RESULT_ERROR_UNSUPPORTED_SIZE =
+        50, ///< [Validation] size argument is not supported by the device
+            ///< (e.g., too large)
+    UR_RESULT_ERROR_UNSUPPORTED_ALIGNMENT =
+        51, ///< [Validation] alignment argument is not supported by the device
+            ///< (e.g., too small)
+    UR_RESULT_ERROR_INVALID_SYNCHRONIZATION_OBJECT =
+        52, ///< [Validation] synchronization object in invalid state
+    UR_RESULT_ERROR_INVALID_ENUMERATION =
+        53, ///< [Validation] enumerator argument is not valid
+    UR_RESULT_ERROR_UNSUPPORTED_ENUMERATION =
+        54, ///< [Validation] enumerator argument is not supported by the device
+    UR_RESULT_ERROR_UNSUPPORTED_IMAGE_FORMAT =
+        55, ///< [Validation] image format is not supported by the device
+    UR_RESULT_ERROR_INVALID_NATIVE_BINARY =
+        56, ///< [Validation] native binary is not supported by the device
+    UR_RESULT_ERROR_INVALID_GLOBAL_NAME =
+        57, ///< [Validation] global variable is not found in the module
+    UR_RESULT_ERROR_INVALID_FUNCTION_NAME =
+        58, ///< [Validation] function name is not found in the module
+    UR_RESULT_ERROR_INVALID_GROUP_SIZE_DIMENSION =
+        59, ///< [Validation] group size dimension is not valid for the kernel
+            ///< or device
+    UR_RESULT_ERROR_INVALID_GLOBAL_WIDTH_DIMENSION =
+        60, ///< [Validation] global width dimension is not valid for the kernel
+            ///< or device
+    UR_RESULT_ERROR_MODULE_UNLINKED =
+        61, ///< [Validation] module with imports needs to be linked before
+            ///< kernels can be created from it.
+    UR_RESULT_ERROR_OVERLAPPING_REGIONS =
+        62, ///< [Validation] copy operations do not support overlapping regions
+            ///< of memory
+    UR_RESULT_ERROR_INVALID_HOST_PTR = 63, ///< Invalid host pointer
+    UR_RESULT_ERROR_INVALID_USM_SIZE = 64, ///< Invalid USM size
+    UR_RESULT_ERROR_OBJECT_ALLOCATION_FAILURE =
+        65, ///< Objection allocation failure
+    UR_RESULT_ERROR_ADAPTER_SPECIFIC =
+        66, ///< An adapter specific warning/error has been reported and can be
+            ///< retrieved via the urGetLastResult entry point.
```

Comment:
Maybe it would be best to exclude this file from formatting - I think it was better formatted before

Reply:
yea, I agree.